tests/timetables/test_interval_timetable.py

```diff
@@ -40,6 +40,31 @@
 HOURLY_TIMEDELTA_TIMETABLE = DeltaDataIntervalTimetable(datetime.timedelta(hours=1))
 HOURLY_RELATIVEDELTA_TIMETABLE = DeltaDataIntervalTimetable(dateutil.relativedelta.relativedelta(hours=1))
 
+CRON_TIMETABLE = CronDataIntervalTimetable("30 21 * * *", TIMEZONE)
+
+@pytest.mark.parametrize(
+    "timetable",
+    [
+        pytest.param(CRON_TIMETABLE, id="cron"),
+    ],
+)
```

Comment:
I don’t think there’s need to parametrize this. This problem can only happen for the cron timetable, not anything else.